source/plugins/go/input/lib/cadvisor.go

```diff
@@ -1074,6 +1074,24 @@ func getContainerCpuMetricItems(metricInfo map[string]interface{}, hostName, met
 						if len(os.Getenv("AZMON_RESOURCE_OPTIMIZATION_ENABLED")) > 0 {
 							telemetryProps["resoureceOptimizationEnabled"] = os.Getenv("AZMON_RESOURCE_OPTIMIZATION_ENABLED")
 						}
+						if len(os.Getenv("AZMON_RETINA_FLOW_LOGS_ENABLED")) > 0 {
+							telemetryProps["networkFlowLogsEnabled"] = os.Getenv("AZMON_RETINA_FLOW_LOGS_ENABLED")
```

Comment:
can we also have consistent naming for this as well. From networkFlowLogsEnabled to "nflEnabled"

Reply:
changed